src/main/java/seedu/address/model/company/Company.java

```diff
@@ -34,6 +35,21 @@ public Company(Name name, Phone phone, Email email, Set<Tag> tags) {
         this.phone = phone;
         this.email = email;
         this.tags.addAll(tags);
+        this.startDate = new Date("2024-01-01");
+        this.deadline = new Date("2024-01-02");
+    }
+
+    /**
+     * This constructor is used for integration with date. Remove this comment and old constructor after completion.
+     */
+    public Company(Name name, Phone phone, Email email, Date startDate, Date deadline, Set<Tag> tags) {
```

Comment:
Would it be more intuitive to name it startDate and endDate rather than deadline